src/vzlogger.cpp

```diff
@@ -397,8 +397,8 @@ int main(int argc, char *argv[]) {
 		return (0);
 	}
 
-	// only logging to file if daemonizing
-	options.logging(options.daemon());
+	// This defines whether the logging threads are started. We always want that, right?
+	options.logging(true);
```

Comment:
I am running vzlogger in foreground on a terminal for testing and still want logging. That's why I patched that. I'll remove this commit from the branch.